package.json

```diff
@@ -1,19 +1,19 @@
 {
-  "name": "loqutor",
-  "version": "0.0.1",
-  "description": "Loqutor, the perfect companion for radialists with no time to lose.",
-  "main": "index.js",
+  "name": "Hawkpost",
+  "version": "9000.0.1",
```

Comment:
@dethos I might agree that Hawkpost is _over 9000_, but is this version number intentional? 😛